CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 Mapbox welcomes participation and contributions from everyone.
 
 ## Unreleased
+
+## Mapbox Navigation SDK 2.4.0-beta.1 - March 11, 2022
+### Changelog
+[Changes between v2.5.0-beta.1 and v2.5.0-beta.1](https://github.com/mapbox/mapbox-navigation-android/compare/v2.4.0-alpha.2...v2.4.0-beta.1)
```

Comment:
```suggestion
[Changes between v2.5.0-alpha.2 and v2.5.0-beta.1](https://github.com/mapbox/mapbox-navigation-android/compare/v2.4.0-alpha.2...v2.4.0-beta.1)
```

Reply:
I think it should be 2.4.0 instead of 2.5.0.

Reply:
nice :)

Reply:
But you still mention 2.5.0 instead of 2.4.0.

Reply:
lol yeah, wake up and make mistakes. it's fixed